src/test/java/org/broadinstitute/hellbender/tools/spark/sv/DiscoverVariantsFromContigAlignmentsSGASparkIntegrationTest.java

```diff
@@ -0,0 +1,154 @@
+            "asm000001:tig00012\t1-195%CTG=21START=27375090END=27375284%195M%+%60%0",
+            "asm000001:tig00013\t1-165%CTG=21START=27375137END=27375301%165M%+%60%0",
+            "asm000001:tig00014\t1-151%CTG=21START=27375223END=27375373%151M%+%60%0",
+            "asm000001:tig00015\t1-151%CTG=21START=27375181END=27375331%151M%+%60%1",
+            "asm000001:tig00016\t1-169%CTG=21START=27375550END=27375718%169M%+%60%0",
+            "asm000001:tig00017\t1-151%CTG=21START=27375487END=27375637%151M%+%60%1"
+    };
+
+    private static final class DiscoverVariantsFromContigAlignmentsSGASparkIntegrationTestArgs {
+        final String inputAssemblies;
+        final String inputAlignments;
+        final String outputDir;
+
+        DiscoverVariantsFromContigAlignmentsSGASparkIntegrationTestArgs(final String inputAssemblies, final String inputAlignments, final String outputDir){
+            this.inputAssemblies = inputAssemblies;
+            this.inputAlignments = inputAlignments;
+            this.outputDir = outputDir;
+        }
+
+        String getCommandLineNoApiKey() {
+            return  " --inputAssemblies " + inputAssemblies +
+                    " --inputAlignments " + inputAlignments +
+                    " --fastaReference " + SVIntegrationTestDataProvider.reference +
+                    " -R " + SVIntegrationTestDataProvider.reference_2bit +
+                    " -O " + outputDir + "/variants.vcf";
+        }
+
+        String getCommandLine() {
+            return  getCommandLineNoApiKey() +
+                    " --apiKey " + getGCPTestApiKey();
+        }
+    }
+
+    @DataProvider(name = "discoverVariantsFromContigAlignmentsSGASparkIntegrationTest")
+    public Object[][] createTestData() throws IOException {
+        List<Object[]> tests = new ArrayList<>();
+
+        final File tempWorkingDir = BaseTest.createTempDir("whatever");
+        tempWorkingDir.deleteOnExit();
+
+        final File assemblyWithPackedFastaWithLength = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"assemblyWithPackedFastaWithLength")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(assemblyWithPackedFastaWithLength, "contents.txt") )  ){
+            out.println( contigs );
+        }
+        final File alignmentsDir = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"alignments")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(alignmentsDir, "alignments.txt") )  ){
+            for(final String aln : alignments) out.println( aln );
+        }
+
+        tests.add(new Object[]{new DiscoverVariantsFromContigAlignmentsSGASparkIntegrationTest.DiscoverVariantsFromContigAlignmentsSGASparkIntegrationTestArgs(
+                assemblyWithPackedFastaWithLength.getAbsolutePath(),
+                alignmentsDir.getAbsolutePath(),
+                tempWorkingDir.getAbsolutePath())});
+
+        return tests.toArray(new Object[][]{});
```

Comment:
Again it seems like it would be straightforward to include the results, what do you think?